src/scss/components/_donors.scss

```diff
@@ -0,0 +1,61 @@
+.donors_info {
+  @include grid;
+}
+
+.donors {
+  position: relative;
+  overflow-x: hidden;
+}
+
+.donors_image {
+  width: 100%;
+}
+
+.donors_description {
+  @include body-xl-med;
+
+  font-size: 2.5rem;
+  @include col(2, 8);
+}
+
+.donors_link_container {
+  @include link-typography;
+  @include col(2, 5);
+
+  display: flex;
+  padding: 2.5rem 0 19.5rem;
+}
+
+.donors_ctaPrimary_link a {
+  color: #fff;
+  border-radius: 100px;
+  padding: 1.25rem 5.4rem;
+  background-color: #08152e;
+  margin-right: 3rem;
+}
+
+.donors_ctaSecondary_link a {
+  position: relative;
+  color: #08152e;
+  padding: 0.75rem 0;
+}
+
+.donors_ctaSecondary_link a::before {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  border-bottom: solid 2px #08152e;
+}
```

Comment:
```suggestion
.donors_link_container {
  @include link-typography;
  @include col(2, 4);
  width: 100%;
  display: flex;
}

.donors_ctaPrimary_link {
  width: 22rem;
  height: 5.6rem;
  border-radius: 5.6rem;
  background-color: var(--brand-navy);
  display: flex;
  justify-content: center;
  align-items: center;
}

.donors_ctaPrimary_link a {
  color: var(--white);
}

.donors_ctaSecondary_link {
  display: flex;
  justify-content: center;
  align-items: center;
  width: 50%;
}

.donors_ctaSecondary_link a {
  color: var(--brand-navy);
  border-bottom: solid 0.2rem var(--brand-navy);
  padding-bottom: 1.2rem;
}
```

Reply:
Thanks for the comment, but when trying it the asymmetrical buttons remain